ibis/common/annotations.py

```diff
@@ -199,25 +229,24 @@ def from_callable(cls, fn, validators=None, return_validator=None):
 
         parameters = []
         for param in sig.parameters.values():
-            if param.kind in {
-                VAR_POSITIONAL,
-                VAR_KEYWORD,
-                POSITIONAL_ONLY,
-                KEYWORD_ONLY,
-            }:
+            if param.kind in {POSITIONAL_ONLY, KEYWORD_ONLY}:
```

Comment:
Should be possible to support both parameter kinds with slight modifications in `unbind()`

Reply:
Added support for both of them.